test/powershell/engine/Help/UpdatableHelpSystem.Tests.ps1

```diff
@@ -0,0 +1,268 @@
+$expectedCompressedFiles["PackageManagement"] = @("PackageManagement_4ae9fd46-338a-459c-8186-07f910774cb8_en-US_helpcontent$extension")
+$expectedCompressedFiles["PowershellGet"] = @("PowershellGet_1d73a601-4a6c-43c5-ba3f-619b18bbb404_en-US_helpcontent$extension")
+$expectedCompressedFiles["PSReadline"] = @("PSReadline_5714753b-2afd-4492-a5fd-01d9e2cff8b5_en-US_helpcontent$extension")
+
+# These are the paths where the help content will be installed to.
+$getHelpContentPaths = @(
+    "$pshome\en-US"
+    "$pshome\Modules\CimCmdlets\en-US"
+    "$pshome\Modules\Microsoft.PowerShell.Archive\en-US"
+    "$pshome\Modules\Microsoft.PowerShell.LocalAccounts\en-US"
+    "$pshome\Modules\PackageManagement\en-US"
+    "$pshome\Modules\PowershellGet\en-US"
+    "$pshome\Modules\PSReadline\en-US"
+)
+    
+function DeleteInboxHelpContent
+{
+    foreach ($path in $getHelpContentPaths)
+    {
+        if (Test-Path $path)
+        {
+            Get-ChildItem $path -Include @("about_*.txt","*help.xml") -Recurse -ea SilentlyContinue | Remove-Item -Force -ErrorAction SilentlyContinue
+        }
+    }
+}
+
+function GetFiles
+{
+    param (
+        [string]$fileType = "*help.xml",
+        [string[]]$path = $getHelpContentPaths
+    )
+                            
+    foreach ($thisPath in $path)
+    {
+        if (Test-Path $thisPath)
+        {
+            return @(Get-ChildItem $thisPath -Include $fileType -Recurse -ea SilentlyContinue | foreach {$_.FullName})
+        }
+    }
+}
+
+function ValidateInstalledHelpContent
+{
+    param ([string]$moduleName)
+
+    $helpFilesInstalled = GetFiles | ForEach-Object {Split-Path $_ -Leaf}
+    $expectedHelpFiles = $expectedHelpInfoFiles[$moduleName]
+
+    for ($index = 0; $index -lt $expectedHelpFiles.Count; $index++)
+    {
+        $expectedHelpFiles[$index] | Should Be $helpFilesInstalled[$index]
+    }
+}
+
+function RunUpdateHelpTests
+{
+    param (
+        [string]$tag = "CI",
+        [switch]$useSourcePath
+    )
+
+    foreach ($moduleName in $moduleList)
+    {
+        It "Validate Update-Help for module '$moduleName'" {
+
+            if ((Get-UICulture).Name -ne "en-Us")
+            {
+                if ($useSourcePath)
+                {
+                    Update-Help -Module $moduleName -Force -UICulture en-US -SourcePath "$PSScriptRoot\assets"
+                }
+                else
+                {
+                    Update-Help -Module $moduleName -Force -UICulture en-US
+                }
+            }
+            else
+            {
+                if ($useSourcePath)
+                {
+                    Update-Help -Module $moduleName -Force -UICulture en-US -SourcePath "$PSScriptRoot\assets"
+                }
+                else
+                {
+                    Update-Help -Module $moduleName -Force
+                }
+            }
```

Comment:
Yes, the culture check is required; otherwise the test will fail in ps-ps. Also, currently, we only have help content on en-US, so we use this to validate update/save-help.